app/views/course/discussion/_posts.html.slim

```diff
@@ -1,7 +1,11 @@
 - post_locals ||= {}
 - posts.each do |post|
   = render partial: post_partial, object: post.first, locals: post_locals
-  div.replies
+  - nest_replies = max_depth > 1 && (posts.each.size > 1 || post.second.length > 1)
```

Comment:
`posts` is actually a PostOrdering object. I probably should define a size method then.